### PR TITLE
Static structure ng

### PR DESCRIPTION
**Pull request by nicolegooden**
## Description

This is a hefty HTML/CSS update, all commits take you from an unstructured HTML and CSS base to a functional mobile layout.  This layout loosely matches two of our wireframe designs, one being the main page with the menu closed and the other being the main page with the menu open.  When the menu is open, all recipe content gets pushed down as planned.  Additionally, there are `.shadow` and `.hidden` classes that handle opacity for the open menu and the initial absence from view, respectively.  None of the HTML elements have the `.shadow` class initially, but this can be added to the desired elements' classList with JS.  All styles in `styles.css` from the top until the media query breakpoint are meant for styling of mobile view, and everything inside of the media query is inclusive of the original CSS that was given to us for desktop.

## Affected areas of application

+ `src/index.html`
+ `src/styles.css`
+ `src/pantry.html`

## How Has This Been Tested?

+ No formal tests for this, but you could check out the screenshots below or open `src/index.html` from the terminal.  I have messed with the size of the browser to simulate a change in screen sizes as well.  It's not perfect yet, but it's a starting point.

## Relevant Tickets

+ None, but maybe we should have added more granular issues relating to HTML and CSS?

## Screenshots

Mobile view with menu closed:

Screen Shot 2020-09-08 at 10 40 09 PM

Mobile view with menu open (`.shadow` added to everything below the menu selections):

Screen Shot 2020-09-08 at 10 42 20 PM

## Questions:

+ If we are creating different pages set in HTML, should each of these files have their own CSS file?

### CODE REVIEW

**Review by CalebCyphers — APPROVED** [review]
This is looking really solid. I think the cards look good, and the menu is looking very very strong. The lines between the menu items look great and the opacity on hover is a really solid interaction. Everything looks nicely centered and clean. I would love to switch the font to one without serifs at some point. 

To your Question: While it is possible to have multiple stylesheets for different pages in the website, ideally we would have only one stylesheet to govern the whole of our site.

To your question: while we can have multiple css documents for multiple pages, ideally we would only have one stylesheet that governed all the pages across the site.

**Comment by nicolegooden** [comment]
Hey @CalebCyphers ! Thanks for your response - yes, changing the font is something on my radar as well! Maybe a comic sans since that is good for accessibility?

Okay, I will start working from the `styles.css` that is already going when I get to `pantry.html` a bit more.  

Also - I should clarify that I don't currently have the opacity enabled on hover...but that is a great idea! As of now, the opacity would take effect when the `.shadow` class from CSS is added to the classList of various elements.

File: src/index.html [hunk 1]
```diff
@@ -4,26 +4,53 @@
...
+        <div class="wrapper">
+          <img class="search-icon" src="./images/search-icon.png" alt="search icon to the left of the search recipes input">
+          <input class="search-recipes-input" type="text" placeholder="search recipes">
+        </div>
+      </section>
+    </section>
+    <main class="main">
+      <section class="all-recipe-cards">
+        <article class="single-recipe-card">
+          <article class="all-card-icons">
```
Comment (taylorjohnson141) [inline src/index.html:43]: This will be were we want to input the fetch request data I think

**Review by taylorjohnson141 — COMMENTED** [review]
Everything looks good!!

**Pull request merged**